src/cli/print/print.go

```diff
@@ -83,16 +84,29 @@ func Only(ctx context.Context, e error) error {
 
 // Err prints the params to cobra's error writer (stdErr by default)
 // if s is nil, prints nothing.
-// Prepends the message with "Error: "
+// You should ideally be using SimpleError or OperationError.
 func Err(ctx context.Context, s ...any) {
 	out(ctx, getRootCmd(ctx).ErrOrStderr(), s...)
 }
 
 // Errf prints the params to cobra's error writer (stdErr by default)
 // if s is nil, prints nothing.
-// Prepends the message with "Error: "
+// You should ideally be using SimpleError or OperationError.
 func Errf(ctx context.Context, tmpl string, s ...any) {
-	outf(ctx, getRootCmd(ctx).ErrOrStderr(), "\nError: \n\t"+tmpl+"\n", s...)
+	outf(ctx, getRootCmd(ctx).ErrOrStderr(), tmpl, s...)
+}
+
+func SimpleError(ctx context.Context, message string) {
+	Err(ctx, color.Red("Error: "+message))
+}
+
+func OperationError(ctx context.Context, header string, body error, metadata map[string]any) {
+	meta, err := json.Marshal(metadata)
+	if err != nil {
+		meta = []byte("Unable to marshal error metadata")
+	}
+
+	Err(ctx, color.Red("Error: "+header)+" \nMessage: %v\nMetadata: %s\n", body, meta)
```

Comment:
Not much a fan of this, for the same principles as discussed with observe.  What you're providing here is basically a template. Plus formatting the verbose error values.  On that note, I wholly disagree with displaying the clues values to the end user.  That's largely debugging metadata and will likely make for a poor and difficult to understand experience.  What's left here without clues values is the following call:

```go
// assumes Errf adds the "Error:" prefix and colors things red.
Errf(ctx, "\nMessage: %v\n", err)
```

Per coloring: I'm a fan of coloring the entire output, as that clearly delineates where the error message content ends and the next non-error content begins.

If it feels like I'm being nitpicky about formatting, its because the only difference between `Info[f]` and `Err[f]` _is_ the formatting.  Othewise the funcs are the same.  Case in point, due to the bug with Err() not adding the "Error" prefix, it is _exactly_ the same as Info().  Any formatting we move out of these two funcs serves to erase their point of existance.

Which, you know, could be fine.  Because it's all templates in the end.  We'd be just as well off with only using Infof() and making various error templates like:
```go
const ErrWithHeader color.Red("Error: %v\nMessage: %v\n")
Infof(ctx, ErrWithHeader, err, header)
```